### PR TITLE
Finishing up server logic and amenities repo

### PR DESCRIPTION
**Pull request by Benntobox**
Seeding script now runs with amenities repo setup, server returns data correctly

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
+app.use(cors())
 
-app.get('/', function (req, res) {
-  console.log('Got request');
-  seeder();
-  res.end();
+app.get('/amenities-api/amenities', function (req, res) {
+  retrieve({})
+    .then(data => {
+      if (data.length === 0) { res.status(400).json("ID not found") }
```
Comment (joshuabradley012) [inline server/server.js]: Error message is misleading here, could respond with "No records found" or something along those lines

**Review by joshuabradley012 — APPROVED** [review]

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const server = require('./server.js');
+const { initialize } = require('./db/index.js')
+require('dotenv').config()
+
+if (process.env.NODE_ENV !== 'test') {
+  initialize();
+  let port = process.env.DEV_PORT || 0;
```
Comment (samasno) [inline server/index.js]: Should have a valid port number as fallback

File: server/server.js [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
-app.get('/show', function (req, res) {
-  retrieve({}) 
-    .then(data => {
-      console.log('Getting all data: ', data.length);
-      res.json(data);
-    });
+app.get('/amenities-api/amenity/:id', function (req, res) {
+  const { id } = req.params;
+  retrieve({ id })
+    .then(data => res.json(data[0]));
```
Comment (samasno) [inline server/server.js]: This line returning `data[0`] rather than `data` created a small break in the nearby-buildings component

**Review by samasno — APPROVED** [review]
Looks good

**Pull request merged**